tools/flaky-test-reporter/result.go

```diff
@@ -130,7 +130,7 @@ func flakyRateAboveThreshold(rd *RepoData) bool {
 // createArtifactForRepo marshals RepoData into json format and stores it in a json file,
 // under local artifacts directory
 func createArtifactForRepo(rd *RepoData) error {
```

Comment:
should this be a method on repoData struct instead of taking it in as an arg? Not related to your change, but can be done later

Reply:
most of the functions in this file could be changed to methods, to be fair. I could do this in another PR.